src/components/header/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Menu } from '../';
+import './styles.css';
+const mainMenu = [
+    { text: 'New',    url: 'https://news.ycombinator.com/newest' },
+    { text: 'Show',   url: 'https://news.ycombinator.com/show' },
+    { text: 'Submit', url: 'https://news.ycombinator.com/submit' },
+  ];
+
+
+
+
+export  const Header = () => (
+    <div className="header">
+        <span className="header__logo">
+        <img src="https://news.ycombinator.com/y18.gif" alt="React news logo"/>
+        React News
+        </span>
+        <div className="header__menu">
+        <Menu links={mainMenu} />
+       </div>
```

Comment:
it will be easier to read if the indentation will be in place. like this:

```jsx
<div className="header">
  <span className="header__logo">
    <img src="https://news.ycombinator.com/y18.gif" alt="React news logo"/>
    React News
  </span>
  <div className="header__menu">
    <Menu links={mainMenu} />
  </div>
</div>
```